fmralign/generate_data.py

```diff
@@ -0,0 +1,153 @@
+"""Some functions to generate toy fMRI data using shared stimulus information."""
```

Comment:
WDYT of putting these two functions inside `fetch_example_data` instead of in their own file ?

Reply:
Done !